tests/conftest.py

```diff
@@ -53,6 +53,12 @@ def gemini_path(request):
     gemini_db = "tests/fixtures/HapMapFew.db"
     return gemini_db
 
+@pytest.fixture(scope='function')
```

Comment:
Sorry I added this then changed my mind but forgot to remove. Thanks for noting!
We should have a dedicated test database as issued in #225